### PR TITLE
Changed os.date() "%c" specifier example format to actual output

### PR DESCRIPTION
**Pull request by Redusofficial**
## Changes

Changes the os.date()  "%c" specifier example to reflect the actual output of it

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

**Comment by Redusofficial** [comment]
The example implies the output of the following code would be "
```lua
print(os.date("%c"))
```

**Comment by Redusofficial** [comment]
The documentation example implies the output of the following code would be "Wed Sep 16 23:48:10 1998". However, the actual output would be formatted as "Wed Sep 16 23:48:10 1998".
```lua
print(os.date("%c"))
```

File: content/en-us/reference/engine/libraries/os.yaml [hunk 1]
```diff
@@ -80,36 +80,128 @@ functions:
...
+      </tr>
       </tbody>
       </table>
 
       \* This value can vary depending on the current locale.
 
-      &dagger; The example provided is for September 16th, 1998 (a Wednesday) at
-      11:48:10 PM (23:48:10), ran using locale "en-us" in Eastern Time (ET).
+      &dagger; The example provided is for February 12th, 2024 (Monday) at
+      2:14:35 PM (14:14:35), run using locale "en-us" in Pacific Time (PST).
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/libraries/os.yaml:204]:
```suggestion
      2:14:35 PM (14:14:35), run using locale "en-us" in Pacific Standard Time (PST).
```

File: content/en-us/reference/engine/libraries/os.yaml [hunk 2]
```diff
@@ -80,36 +80,128 @@ functions:
...
+      </tr>
+      <tr>
+        <td>%Y</td>
+        <td>Full year</td>
+        <td>2024</td>
+      </tr>
+      <tr>
+        <td>%z</td>
+        <td>ISO 8601 offset from UTC in timezone (1 minute = 1, 1 hour = 100)</td>
+        <td>-1000</td>
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/libraries/os.yaml:186]:
```suggestion
        <td>-0800</td>
```

**Comment by IgnisRBX** [comment]
I updated the entire table to use a more modern date and match the exact format that Studio produces. If you'd like to confirm that these numbers all look correct, I'll merge. Thanks for the contribution!

**Comment by Redusofficial** [comment]
All looks good.

**Comment by Redusofficial** [comment]
> I updated the entire table to use a more modern date and match the exact format that Studio produces. If you'd like to confirm that these numbers all look correct, I'll merge. Thanks for the contribution!
All looks good.

**Review by IgnisRBX — APPROVED** [review]

**Pull request merged**